src/core/stdc/stdio.d

```diff
@@ -922,6 +922,8 @@ else version( CRuntime_DigitalMars )
     pure int  feof(FILE* stream)     { return stream._flag&_IOEOF;                       }
     ///
     pure int  ferror(FILE* stream)   { return stream._flag&_IOERR;                       }
+    ///
```

Comment:
Redoing core headers like this should be in a separate PR. Can you please pull the stdio.d and windows.d out into separate PRs, and let me know so I can pull them for you?